app/views/users/webauthn_setup/new.html.erb

```diff
@@ -33,20 +38,38 @@
 
   <%= hidden_field_tag :platform_authenticator, @platform_authenticator, id: 'platform_authenticator' %>
   <% if !@platform_authenticator %>
-    
-      <%= render ValidatedFieldComponent.new(
-            form: f,
-            name: :name,
-            required: true,
-            label: @presenter.nickname_label,
-            hint: @presenter.device_nickname_hint,
-            input_html: {
-              id: 'nickname',
-              class: 'font-family-mono',
-              size: 16,
-              maxlength: 20,
-            },
-          ) %>
+    <fieldset aria-labelledby="webauthn-intro" class="padding-0 border-0 margin-y-4 margin-x-0">
```

Comment:
This is a rather long label for the fieldset. At the very least I might suggest excluding the link from the label since it's not relevant for how it's used by a screen reader.

![image](https://github.com/18F/identity-idp/assets/1779930/5c759db8-5ed7-4a17-9521-6e9904935bf6)

Reply:
Removed the link from the label.